composer.json

```diff
@@ -41,17 +41,13 @@
             },
```

Comment:
@mibstar I would not do that. It's going to get tricky to maintain the branch.

I would just explain textually in a `.md` document the changes that someone needs to do to use the latest -dev version of Drupal core.

Does that make sense?